localisation/00_zuwc_united_localisation.csv

```diff
@@ -3024,11 +3004,29 @@ RULE_ADOPTION_NO_PLAYER_DYNASTY;Restricted;;;;;;;;;;;;;x
 RULE_ADOPTION_NO_PLAYER_DYNASTY_DESC;Only players can adopt newly-spawned characters by adoption event into the player's dynasty.;;;;;;;;;;;;;x
 RULE_ADOPTION_DISABLED;Disabled;;;;;;;;;;;;;x
 RULE_ADOPTION_DISABLED_DESC;Adoption event is disabled.;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+faction_action_cooldown;Faction Action Cooldown;;;;;;;;;;;;;x
+EVTDESC_WCPLF_106;[Root.GetTitledFirstNameWithNick], the current leader of the [e_horde.GetFullBaseName], is no longer interested and can�t afford supporting the ones who decided to join the [e_horde.GetFullBaseName]'s side.\n\n[Root.GetSheHeCap] has announced the disbanding of the political faction. All emissaries have been recalled and former allies have been left without the [e_horde.GetFullBaseName]'s military support.;;;;;;;;;;;;;x
+GetLeaveDisbandCap_Disband;Disband;;;;;;;;;;;;;x
+GetLeaveDisbandCap_Leave;Leave;;;;;;;;;;;;;x
+GetLeaveDisband_disband;disband;;;;;;;;;;;;;x
+GetLeaveDisband_leave;leave;;;;;;;;;;;;;x
+found_horde_political_faction;Create the Horde Faction;;;;;;;;;;;;;x
+found_alliance_political_faction;Create the Alliance Faction;;;;;;;;;;;;;x
+found_horde_political_faction_desc;You create the Horde Faction, so independent rulers can join your side as an ally or even a vassal.;;;;;;;;;;;;;x
+found_alliance_political_faction_desc;You create the Alliance Faction, so independent rulers can join your side as an ally or even a vassal.;;;;;;;;;;;;;x
+join_in_horde_political_faction;Join the Horde Faction;;;;;;;;;;;;;x
+join_in_alliance_political_faction;Join the Alliance Faction;;;;;;;;;;;;;x
+join_in_horde_political_faction_desc;You send a letter with the desire to join the Horde Faction.;;;;;;;;;;;;;x
+join_in_alliance_political_faction_desc;You send a letter with the desire to join the Alliance Faction.;;;;;;;;;;;;;x
+leave_horde_political_faction;[Root.GetLeaveDisbandCap] the Horde Faction;;;;;;;;;;;;;x
+leave_alliance_political_faction;[Root.GetLeaveDisbandCap] the Alliance Faction;;;;;;;;;;;;;x
+leave_horde_political_faction_desc;You [Root.GetLeaveDisband] the Horde Faction, losing the protection and assistance of your former allies. If you're a leader of the Horde, you do NOT lose the Horde title.;;;;;;;;;;;;;x
+leave_alliance_political_faction_desc;You [Root.GetLeaveDisband] the Alliance Faction, losing the protection and assistance of your former allies.;;;;;;;;;;;;;x
```

Comment:
I'm assuming the second sentence was forgotten looking at the former line 👍 
```suggestion
leave_alliance_political_faction_desc;You [Root.GetLeaveDisband] the Alliance faction, losing the protection and assistance of your allies. If you're the leader of the Alliance, you do NOT lose the Alliance title.;;;;;;;;;;;;;x
```

Reply:
Done!